src/main/java/com/spotify/sparkey/Util.java

```diff
@@ -270,4 +270,32 @@ static void nonThrowingClose(Closeable stream) {
       e.printStackTrace();
     }
   }
+
+  static boolean renameFile(File srcFile, File destFile) {
+    if (!destFile.exists()) {
+      return srcFile.renameTo(destFile);
+    }
+
+    File backupFile = new File(destFile.getParent(), destFile.getName() + "-backup" + System.currentTimeMillis());
+    if (backupFile.exists()) {
+      // Edge case, but let's be safe.
+      return false;
+    }
+    if (!destFile.renameTo(backupFile)) {
+      return false;
+    }
+    if (destFile.exists()) {
+      // This is strange, it should be renamed by now. But let's be safe.
+      return false;
```

Comment:
Minor comment:  It would be good to have logs or exceptions for failure cases.. mentioning specific issue so that it eases troubleshooting.